playbooks/bootstrap.yml

```diff
@@ -15,6 +15,9 @@
   any_errors_fatal: "{{ edpm_any_errors_fatal | default(true) }}"
   max_fail_percentage: "{{ edpm_max_fail_percentage | default(0) }}"
   tasks:
+    - name: Gather ansible_local facts
+      ansible.builtin.setup:
+        filter: ansible_local
```

Comment:
Isn't using `filter`  would be expensive as it collects all facts and then filters `local`? `gather_subset` would be more efficient everywhere I guess.

```
 gather_subset:
      - "!all"
      - "!min"
      - "local"
```

Reply:
Hmm.. We don't gather facts by default https://github.com/openstack-k8s-operators/edpm-ansible/blob/main/playbooks/bootstrap.yml#L5

Also, this additional task has been added for every role and would be running for non bootc deployments as well and the it takes around 12 secs for a single node in the bootstrap service. My point is this kind of negates the work we had done to disable gather facts by default in edpm. 

TASK [Gather ansible_local facts filter=ansible_local] *************************
Tuesday 16 September 2025  16:09:57 +0000 (0:00:11.483)       0:00:11.514 ***** 
[0;32mok: [edpm-compute-0][0m

Reply:
ok, i updated to use:

```
 gather_subset:
      - "!all"
      - "!min"
      - "local"
```

everywhere